client/src/components/Sidebar/Sidebar.test.js

```diff
@@ -91,7 +91,7 @@ describe('workbench sidebar', () => {
   it.each(DEBUG_PAGES)('is available in debug mode', page => {
     const { getByText } = renderSideBar(
       mockStore({
-        workbench: { status: { debug: true } },
+        workbench: { status: { debug: true }, config: { hideApps: false } },
         notifications,
         ticketCreate
       }),
```

Comment:
@jmartinez-tacc Can we add a unit test here to check that the Apps and History sidebar sections do not render if `hideApps` is `true`?